env.default

```diff
@@ -33,7 +33,7 @@ AWS_SQS_REGION=
 # CSP config
 
 CSP_DEFAULT_SRC="'none'"
-CSP_SCRIPT_SRC='self' 'unsafe-inline' cdn.optimizely.com https://www.google-analytics.com/analytics.js
+CSP_SCRIPT_SRC='self' 'unsafe-inline' cdn.optimizely.com https://www.google-analytics.com/analytics.js https://c.shpg.org/352/sp.js
```

Comment:
I know it's not related to your PR but can you a follow-up bug to make sure we figure out how to remove `unsafe-inline` and still have it work, especially since we're now collecting user data

Reply:
I think unsafe-inline is there for optimizely, and not share progress.

And there isn't a way currently to make optimizely work without unsafe-inline unless we find a replacement to optimizely.

Reply:
Are we using optimizely on the foundation site? (We probably need to in the future, but I just don't think that's what the `unsafe-inline` is for at this point)